blog/_posts/2020-06-18-the-right-display-for-now.md

```diff
@@ -0,0 +1,73 @@
+---
+slug: "the-right-display-for-now"
+date: 2020-06-18
+author: nick
+layout: blog
+title: "The Right Display for Now"
+image: "/assets/images/the-right-display-for-now/forest-park.jpg"
+tags:
+  - tilt
+  - ui
+---
+
+At Tilt, Han and I talk a lot about the different modes developers are in and
+problems they're solving when using software. Sometimes it's easier to talk
+about these modes when we relate them to signs in the real world.
+
+For example:
+
+- Discovery - what is it possible to do here? (see: the Forest Park sign above)
+- Awareness - what issues should I be aware of? (see: the estimated arrival time signs at train stations)
+- Recommendations - what should I do next? (see: exit signs, recommended routes)
+
+Tilt has always initialized your terminal with a terminal-based GUI. 
+
+We've found that the terminal-based GUI is good for awareness of all your
+different services. But as more teams adopt Tilt, it does a worse and worse job
```

Comment:
@nicks : What does this mean? Do you mean more teams within a company? As more teams adopt Tilt within a company, why does it do a worse job? If you mean overall, why do you think as more companies/users adopt Tilt, it does a worse job?

Reply:
ya, agreed this is confusing. changed this

Reply:
New version looks great.